neurodamus/core/nodeset.py

```diff
@@ -221,7 +221,7 @@ def add_gids(self, gids, gid_info=None):
         """Add raw gids, recomputing gid offsets as needed"""
         self._gidvec.extend(gids)
         if len(gids) > 0:
-            self._max_gid = max(self.max_gid, max(gids))
+            self._max_gid = max(self.max_gid, np.max(gids))
```

Comment:
is this for efficiency?

Reply:
either that, or I have to use disable the lint; using the ringtest, I put a break on this, and it was of type np.array, so I think it's safe.  The `max(max.gids, *gids)` alternative seems less safe, and less performant.